eth_tester/exceptions.py

```diff
@@ -28,3 +28,11 @@ class AccountLocked(Exception):
 
 class TransactionFailed(Exception):
     pass
+
+
+class Revert(Exception):
```

Comment:
Should these be subclasses of `TransactionFailed`?  Or maybe just use `TransactionFailed` anyplace that you are currently using `Revert`?  I think my preference is for the later.